yoti/src/Plugin/Block/YotiBlock.php

```diff
@@ -34,63 +35,42 @@ public function build() {
     $user = \Drupal::currentUser();
 
     // No config? no button.
-    $config = YotiHelper::getConfig();
-    if (!$config) {
+    $config = \Drupal::service('yoti.config');
+    if (!$config->getSettings()) {
       return [];
     }
 
-    $script = [];
-
     // If connect url starts with 'https://staging' then we are in staging mode.
```

Comment:
Minor point - do you think this line should go below the two variable declarations below? (so it's just above the bit of code it's referring to)